src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let sum = days * 40;
```

Comment:
Consider using `const` instead of `let` for `sum` since it is not being reassigned, to ensure the value doesn't get changed unintentionally.